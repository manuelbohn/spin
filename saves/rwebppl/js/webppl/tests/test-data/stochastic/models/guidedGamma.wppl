var model = function() {
  sample(Gamma({shape: 12, scale: 1 / 8}));
};
