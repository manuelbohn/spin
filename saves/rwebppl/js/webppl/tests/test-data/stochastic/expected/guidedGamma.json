{
  "mean": 1.5,
  "std": 0.433
}
